src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    public String [] calculating(StringBuilder builder) {
+        String [] separateLine = builder.toString().split(System.lineSeparator());
+        int [] result = new int[] {0, 0};
```

Comment:
```suggestion
        String[] separateLine = builder.toString().split(System.lineSeparator());
        int[] result = new int[] {0, 0};
```
remove whitespaces everywhere